### ISSUE TITLE
just a minor bug fix?

### ISSUE THREAD

**Issue by resigned** [crock/donate-bot#1]
Hey, I noticed in index.js you used indexOf. Won't that always trigger if the character is in the string?

**Comment by resigned** [comment]
Yes however it will return true  if there's a ! anywhere in the message.

**Comment by crock** [comment]
I just tested this by posting a regular message containing a exclamation point at the the end to my server that contains one of my custom bots with the above code. There is no adverse effect.

**Comment by crock** [comment]
In other words, it is supposed to return the message if it is not a command. A return statement breaks out of the function.
